qmxgraph/api.py

```diff
@@ -233,6 +276,15 @@ def remove_cells(self, cell_ids):
         """
         return self.call_api('removeCells', cell_ids)
 
+    def remove_port(self, vertex_id, port_name):
+        """
+        Remove an existing port from a vertex. Any edge connected to the
+        vertex through the port is also removed.
+
+        :param str vertex_id: The id of the parent vertex.
+        :param str port_name: The port's name to remove.
+        """
```

Comment:
Perhaps we can add a test which at least ensures all api functions are covered? It can even test that the correct parameters are being passed.